src/protocolgame.cpp

```diff
@@ -3315,7 +3317,7 @@ void ProtocolGame::sendVIPEntries()
 	for (const VIPEntry& entry : vipEntries) {
 		VipStatus_t vipStatus = VIPSTATUS_ONLINE;
 
-		Player* vipPlayer = g_game.getPlayerByGUID(entry.guid);
+		auto vipPlayer = g_game.getPlayerByGUID(entry.guid);
```

Comment:
Could this be `const auto`
```suggestion
		const auto vipPlayer = g_game.getPlayerByGUID(entry.guid);
```